src/main/webapp/dev/js/main/feedbackSubmissionsEdit.js

```diff
@@ -525,7 +534,7 @@ function updateConstSumMessageQn(qnNum) {
             messageElement.removeClass('text-color-red');
             messageElement.removeClass('text-color-green');
         } else if (remainingPoints === 0) {
-            if (!forceUnevenDistribution || allUnique) {
+            if (!forceUnevenDistribution || allUnique || (forceSomeUnevenDistribution && !allSame)) {
```

Comment:
I noticed that the condition u added corresponds to the case of "distributing some points unevenly". I like this combination of the conditions, it specifies the case (forceSomeUnevenDistribution) and the constraint(!allSame). I think you can do the same thing to the other two conditions i.e. `!forceUnvevenDistribution` and `allUnique`